.gitignore

```diff
@@ -40,3 +40,5 @@ testem.log
 # System files
 .DS_Store
 Thumbs.db
+
+server-api/node_modules
```

Comment:
* install editor.config vscode extension - which will ensure correct formatting and indentation is followed for each file
* this will also add a line at the end of each file - note the red icon after line 44 above

Reply:
Okay, Installed extension and added the line